completion_aggregator/tasks/aggregation_tasks.py

```diff
@@ -245,6 +293,43 @@ def _aggregator_needs_update(self, block, modified, force):
             return getattr(agg, 'last_modified', OLD_DATETIME) < modified
         return False
 
+    def _is_aggregator_data_valid(self, user, course_key, block_key):
```

Comment:
Okay, just saw your comment in the description. It does not need to be converted to a class method since a manager instance is available at `Aggregator.objects`.